docs/pages/includes/port-table.mdx

```diff
@@ -0,0 +1,78 @@
+Teleport services listen on several ports. This table shows the default port numbers.
+
+<table>
+  <thead>
+    <tr>
+      <td>Port</td>
+      <td>Service</td>
+      <td>Description</td>
+      <td>Ingress</td>
+      <td>Egress</td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>443</td>
+      <td>Proxy</td>
+      <td>Default proxy port when using ACME for TLS/SSL.</td>
+      <td>Allow inbound SSL/TLS connections to the proxy host.</td>
+      <td>-</td>
+    </tr>
+    <tr>
+      <td>3022</td>
+      <td>Node</td>
+      <td>SSH port to the Node Service. This is Teleport's equivalent of port `22` for SSH.</td>
+      <td>Allow inbound traffic from the proxy host.</td>
+      <td>Allow outbound traffic to the proxy host.</td>
+    </tr>
+    <tr>
+      <td>3023</td>
+      <td>Proxy</td>
+      <td>SSH port clients connect to after authentication. A proxy will forward this connection to port `3022` on the destination node.</td>
+      <td>Allow inbound traffic from SSH clients.</td>
+      <td>Allow outbound traffic to SSH clients.</td>
+    </tr>
+    <tr>
+      <td>3024</td>
+      <td>Proxy</td>
+      <td>SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server.</td>
+      <td>Alllow inbound traffic when behind a firewall.</td>
+      <td>-</td>
+    </tr>
+    <tr>
+      <td>3025</td>
+      <td>Auth</td>
+      <td>SSH port used by the Auth Service to serve its Auth API to other nodes in a cluster.</td>
```

Comment:
`3025` is using TLS protocol, not SSH

Reply:
Corrected. My bad. Looks like we have that up presently.